modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -419,27 +436,26 @@ export default class NestedTabs extends Base {
 	}
 
 	updateIndexValues() {
-		const { $tabContents, $tabTitles } = this.getDefaultElements(),
+		const { $widgetContainer, $tabContents, $tabTitles } = this.getDefaultElements(),
 			settings = this.getSettings(),
-			itemIdBase = $tabTitles[ 0 ].getAttribute( 'id' ).slice( 0, -1 ),
```

Comment:
The previous approach had issues when we would 10 or more tabs.
In my approach I am using the original` widgetNumber`, so that we always create a correct ID.